### DISCUSSION TITLE
I'm upgradin librdkafka version to 1.9.2, compatibility  with openssl - 3.0.7 version?

### DISCUSSION THREAD

**Discussion by IM2555** [edenhill/librdkafka#4073] [Q&A]
I'm planning to upgrade librdkafka version to 1.9.2 and will be building with openssl - 3.0.7  statically.  just wanted to check if librdkafka version 1.9.2 is compatible with openssl  3.0.7

**Comment by edenhill** [comment]
OpenSSL 3 is not yet supported, but it will be in the upcoming release of librdkafka (end of december).

**Reply by IM2555** [reply]
@edenhill Could you please let me know, when can we expect new librdkafka version compatible with openssl 3.0.7
